h2/src/docsrc/help/help.csv

```diff
@@ -855,6 +855,8 @@ work. Linked tables to the same database share one connection.
 
 READONLY - is set, the remote table may not be updated. This is enforced by H2.
 
+FETCH_SIZE - the number of rows fetched
```

Comment:
Such description may confuse people. You can write that specified value will be passed as a hint to the `""java.sql.Statement.setFetchSize()""` method.

Reply:
done